layouts/partials/footer.html

```diff
@@ -4,5 +4,6 @@
       <p>{{.}}</p>
     {{ end }}
     {{ if not .Site.Params.hideCopyright }} © {{ .Site.LastChange.Format "2006" }} {{ end }} {{ if not .Site.Params.hideCredits}} {{ if not .Site.Params.hideCopyright }} · {{ end }} Powered by <a href="https://gohugo.io/">Hugo</a> & <a href="https://github.com/luizdepra/hugo-coder/">Coder</a>. {{ end }}
+    {{ with .Site.Params.commit }}&nbsp;&bull;&nbsp;[<a href="{{ . }}{{ getenv "GIT_COMMIT_SHA" }}">{{ getenv "GIT_COMMIT_SHA_SHORT" }}</a>]{{ end }}
```

Comment:
Just one minor fix.
Replace these 2 last lines with the following code:
```
{{ if not .Site.Params.hideCopyright }} © {{ .Site.LastChange.Format "2006" }}{{ end }}
    {{ if not .Site.Params.hideCredits }}{{ if not .Site.Params.hideCopyright }} · {{ end }}Powered by <a href="https://gohugo.io/">Hugo</a> & <a href="https://github.com/luizdepra/hugo-coder/">Coder</a>.{{ end }}
    {{ if .Site.Params.commit }}{{ if or (not .Site.Params.hideCredits) (not .Site.Params.hideCopyright) }} · {{ end }}[<a href="{{ .Site.Params.commit }}{{ getenv "GIT_COMMIT_SHA" }}">{{ getenv "GIT_COMMIT_SHA_SHORT" }}</a>]{{ end }}
```
This way the theme can handle any combination of `commit`, `hideCopyright` and `hideCredits`.

Reply:
I've got this change fixed.